src/types.ts

```diff
@@ -2,4 +2,10 @@ export interface BodyWeightType {
     id: number,
     date: string,
     weight: string,
+}
+
+export interface Notification {
+    notify: boolean
+    message: string
+    success: boolean
```

Comment:
If we use the alert component, we could pass the type here (error, warning, info, success)

Reply:
Ok 👍🏽